content/06.prior-knowledge.md

```diff
@@ -1,27 +1,26 @@
+They augmented their text dataset by mapping related clinical terms together - e.g., mapping "cancer" and "malignancy" in different patients to the same clinical concept.
+With this enhanced dataset, they trained and tested a variety of text classification algorithms to identify rare disease patients within their corpus. (Figure [@fig:4]b, Box 1a)
 
-Rare disease researchers have also integrated multiple KGs and applied neural network-based algorithms optimized for graph data, such as a graph convolutional neural network.
-Rao and colleagues [@doi:10.1186/s12920-018-0372-8] describe the construction of a KG using phenotype information (Human Phenotype Ontology) and rare disease information (Orphanet) and curated gene interaction/pathway data (Lit-BM-13, WikiPathways) [@pmc:PMC7778952; @doi:10.1016/j.cell.2014.10.050; @doi:10.1093/nar/gkaa1024]. 
-They then trained a spectral graph convolution neural network on this KG to identify and rank potentially causal genes for the rare diseases from Orphanet, and were able to use this model to accurately predict causal genes for a ground truth dataset of rare diseases with known causal genes.
+Rare disease researchers have also integrated multiple KGs and applied neural network-based algorithms optimized for graph data, such as a graph convolutional neural networks.
+Rao and colleagues [@doi:10.1186/s12920-018-0372-8] describe the construction of a KG using phenotype information from the Human Phenotype Ontology, and rare disease information from Orphanet and curated gene interaction/pathway data from Lit-BM-13 and WikiPathways [@pmc:PMC7778952; @doi:10.1016/j.cell.2014.10.050; @doi:10.1093/nar/gkaa1024]. 
+They trained a spectral graph convolution neural network on this KG to identify and rank potentially causal genes for the rare diseases from Orphanet, and used this model to accurately predict causal genes for a ground truth dataset of rare diseases with known causal genes.
 While several groups have used KGs to study rare diseases, we expect that better multi-modal datasets and ML methods to analyze KGs will make them a more popular and important tool in rare disease. 
 
-Another approach that builds on prior knowledge and large volumes of related data is transfer learning.
-Transfer learning leverages shared features, e.g., normal developmental processes that are aberrant in disease or an imaging anomaly present in both rare and common diseases, to advance our understanding of rare diseases. 
+Another approach that builds on prior knowledge and large volumes of related data is transfer learning, a modeling technique that “borrows strength” across datasets with both similar and distinct properties like normal developmental processes that are aberrant in disease or an imaging anomaly present in both rare and common diseases, to advance our understanding of rare diseases. 
```

Comment:
re: `normal developmental processes that are aberrant in disease`
Justin: This example doesn’t make sense

Reply:
```suggestion
Another approach that builds on prior knowledge and large volumes of related data is transfer learning, a modeling technique that “borrows strength” across datasets with both similar and distinct properties, e.g. an imaging anomaly present in both rare and common diseases, to advance our understanding of rare diseases. 
```